src/Chirp.Razor/Pages/Public.cshtml

```diff
@@ -1,18 +1,19 @@
-﻿@page "/"
+﻿@page "/page"
```

Comment:
This should probably still be the front page.

Reply:
Yeah for sure! Fixed 71a789b5b6b3b00935442835cd89b51b553a1ab4